views/index.html

```diff
@@ -16,15 +16,11 @@
     <script src="/script.js" defer></script>
   </head>
   <body>
-    <header>
-      <h1>HTML-only chat</h1>
-    </header>
-
-    <main>
-      <iframe src="/messages"></iframe>
-      <form action="/messages" method="post">
-        <input name="message" type="text" />
-        <button type="submit">submit</button>
+    <main class="c-chat">
+      <iframe class="c-chat__view" src="/messages#latest"></iframe>
+      <form class="c-chat__chat-form" action="/messages" method="post" autocomplete="off">
+        <input class="c-chat__message-input" autofocus name="message" type="text" />
```

Comment:
Turned autofocus for the chat input on. So we don't have to keep clicking on it after a message.

Reply:
Great idea!